### ISSUE TITLE
[Feature Request] Set elevation action: suppress collision warnings & confirm movement option

### ISSUE THREAD

**Issue by fedeicodip** [ironmonk88/monks-active-tiles#838]
In my games moving trough levels becomes quite common so I use the **Set elevation** action a lot.
It would be wonderfull if you could add some minor option to choose to this action that could change its default behaviour:

1. To be able to **suppress collisions** with roofs (in the level addon you can add a special teleport flag to tell levels to skip the collision check): `tokenDocument.update({ elevation: newElevation}, {teleport: true}); `
2. If you want to prompt a message to confirm the movement.
3. At the moment I could not add or subtract height with + and - sign. I was only able to change the triggering token level to a given number. It would be nice to make it add or subtract to the current level of the triggering token (as described in the action notes).

I was able to manage something using the **run code** action and with the following code, this suppresses the collisions warning, but sometimes it doesn't work as expected on the connected player computer (it does simply doesn't apply):

`let tokens = canvas.tokens.controlled;
if(tokens) {
	tokens.forEach(token => {
		token.document.update({elevation: 30 }, {teleport: true})
	});
}`

**Comment by Crowguard** [comment]
2. You can already add a prompt for anything with the Show Dialog action.

**Comment by fedeicodip** [comment]
True, and the show dialog works in a more general way. It’s perfect for that. Just the _suppress collision_ is the option I wish would be there.
